Buttplug.Apps.DeviceSimulatorGUI/Buttplug.Apps.DeviceSimulatorGUI.csproj

```diff
@@ -54,6 +54,9 @@
     <Reference Include="System.Xaml">
       <RequiredTargetFramework>4.0</RequiredTargetFramework>
     </Reference>
+    <Reference Include="TAlex.WPF.Controls, Version=2.0.1.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
In the future, can you divide out Simulator patches from the rest of the patches? I don't think anyone other than you is using this, and we're not shipping it, so it just kinda gets in the way of reviews.

Reply:
The update to the simulator here was as a testbed for the changes: visualising how a multi-vibrator device works with clients sending SingleMotorVibrateCmd and comparing the speed of the linear actuator when send FleshlightLaunch commands vs LinearCmd.

Without these changes, I couldn't have implemented the changes in the rest of the system. Granted, if the device manager was truely modular, we could shift this into a different repo. Or I could have rebased the changes out temporarily.

Reply:
This is more about patch ordering, not content. On most projects I work on, we rebase/squash feature portions into a single commits, and PR a stack of those commits. So, for instance, if I were doing this PR it would probably be 3 commits when pushed for review: 
- The core changes
- The app changes
- The simulator changes

The way things are in this PR, I basically have 1 1.5k LOC, 42 file commit to look at, and it's hard to reason about the parts of that separately, especially parts I haven't really been keeping up with like the simulator.

Reply:
That makes sense.
I can give the squash and split a go tomorrow.

Reply:
This one's done.